apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/businessrules/PageRuleController.java

```diff
@@ -0,0 +1,29 @@
+package gov.cdc.nbs.questionbank.businessrules;
+
+
+import gov.cdc.nbs.questionbank.businessrules.response.CreateRuleResponse;
+import gov.cdc.nbs.questionbank.model.CreateRuleRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@RestController
+@RequestMapping("/api/v1")
+public class PageRuleController {
+
+    @Autowired
+    private PageRuleService pageRuleService;
+
+    @PreAuthorize("hasAuthority('LDFADMINISTRATION-SYSTEM')")
+    @RequestMapping(method = RequestMethod.POST, value = "/createPageRule", consumes = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
```

Comment:
can use `PostMapping` here. And we'd like to follow [REST URI naming best practices](https://restfulapi.net/resource-naming/).

So the creation of a rule would be a `POST` to `page-rule` or just `rule`